### ISSUE TITLE
dxf import entities

### ISSUE THREAD

**Issue by rurban** [LibreDWG/libredwg#142]
The tables, the NOD (named object dict) are not imported and the ACAD APPID not, and the entities neither.
The problem is for sure the HANDLE table (object map), but the OBJECTS table also.
Also the header CRC is wrong.

This is the log of the acad2018 audit:
```
Scanning completed.
Reconstructing handle table from objects.
Scanning for objects  99% done
 0      objects found.
Scanning for objects completed.
Recover: error eDwgCRCDoesNotMatch reading Header.
Missing AcDbRegAppTable
Recover: error 15 opening Named Objects Dictionary.
Created new NOD.
Missing AcDbLinetypeTable
Missing AcDbLayerTable
Missing AcDbBlockTable
Missing AcDbTextStyleTable
Missing AcDbViewTable
Missing AcDbUCSTable
Missing AcDbViewportTable
Missing AcDbDimStyleTable
Missing AcDbVXTable
    Salvaged database from drawing.
Auditing Header
Auditing Tables
Auditing Entities Pass 1
Pass 1 100     objects audited
Auditing Entities Pass 2
RegApp                       ACAD missing
Pass 2 100     objects audited
Auditing Blocks
 0       Blocks audited
Auditing AcDsRecords
Total errors found 1 fixed 1
Erased 0 objects
```

Teigha says:
OdError thrown during readFile of drawing "test/example_2000.dwg" :
  Object improperly read

**Comment by rurban** [comment]
remaining diffs:
* ENDBLK and SEQEND have both NULL prev_entity and next_entity
* color.index: 0 [CMC.BS 62] missing
* SEQEND.plotstyle: NULL 5 [H 390] missing. plotstyle_flags: 0 <=> 3 ??
* size and bitsize

**Comment by rurban** [comment]
Fixed bitsize with 6eca2ada
